packages/@uppy/status-bar/src/locale.ts

```diff
@@ -45,5 +47,5 @@ export default {
       1: '%{smart_count} more files added',
     },
     showErrorDetails: 'Show error details',
-  },
-}
+  } as Locale<0 | 1>['strings'],
```

Comment:
not sure what's the purpose of this

Reply:
It's to force TS to validate the object to be compatible with the defined type.